src/components/FilterBar.jsx

```diff
@@ -30,38 +30,40 @@ const grades = {
   16: 'College Senior',
 };
 
-const sortingOptions = {
-  deadlineSoon: 'Deadline (Earliest to Latest)',
-  deadlineLatest: 'Deadline (Latest to Earliest)',
-  amountLow: 'Amount (Low to High)',
-  amountHigh: 'Amount (High to Low)',
+const sortItems = {
+  'deadline.asc': 'Deadline (Earliest to Latest)',
+  'deadline.desc': 'Deadline (Latest to Earliest)',
+  'amount.asc': 'Amount (Low to High)',
+  'amount.desc': 'Amount (High to Low)',
+};
```

Comment:
Not sure if you wanted to move the descriptions into `sortOptions` and then define `sortItems` using [Object.fromEntries](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Object/fromEntries) like in the [stack overflow example](https://stackoverflow.com/a/14810722) we found earlier.

Feel free to punt this change to a separate PR though